tests/BUILD.bazel

```diff
@@ -88,3 +88,18 @@ go_binary(
     name = "go-test",
     srcs = ["go-test.go"]
 )
+
+# Note that Nix needs to write to /dev/pts (see
+# https://github.com/NixOS/nix/blob/4a2a45f53d16040f744580747d0963f97ec51164/src/libstore/build.cc#L2255)
+# This test has then to be executed with the flag --sandbox_writable_path /dev/pts
```

Comment:
```suggestion
```
As I understand this is no longer true. At least the test passes with a regular `nix-shell --pure --run 'bazel test //...'` for me.

Reply:
Correct. Fixed.

Reply:
Thank you!